app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.user = current_user
+
+    @comment.save if @comment.valid?
+    redirect_to post_path(@comment.post)
```

Comment:
- [ ] I get this error after creating a comment. The comment is actually created but your redirection is incorrect. It should be `posts_path(...)`, not `post_path(...)`.

  ![rurir](https://user-images.githubusercontent.com/54848363/205399911-7a0aa42d-ff0d-48a4-a7c4-281024f1ab2e.jpg)